src/main/java/no/nav/familie/integrasjoner/client/rest/ModiaContextHolderClient.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.integrasjoner.client.rest
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.integrasjoner.config.incrementLoggFeil
+import no.nav.familie.integrasjoner.modiacontextholder.domene.ModiaContextHolderRequest
+import no.nav.familie.integrasjoner.modiacontextholder.domene.ModiaContextHolderResponse
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestOperations
+import java.net.URI
+
+@Component
+class ModiaContextHolderClient(
+    @Value("\${MODIA_CONTEXT_HOLDER_URL}") private val baseURI: URI,
+    @Qualifier("jwtBearer") restTemplate: RestOperations,
+) : AbstractRestClient(restTemplate, "modia-context-holder") {
+    private val contextUri = URI("$baseURI/api/context")
+
+    fun hentContext(): ModiaContextHolderResponse =
+        try {
+            getForEntity(contextUri)
+        } catch (e: Exception) {
+            incrementLoggFeil("modia-context-holder")
+            throw e
```

Comment:
I stedet for å telle manuelt og kaste exception, så bør du kaste OppslagException med level MEDIUM.
Ellers så blir de fanget av den generelle.

Da et det OppslagsException som teller for deg